iXBRLViewerPlugin/viewer/src/js/inspector.js

```diff
@@ -473,7 +474,11 @@ Inspector.prototype._updateValue = function (text, showAll, context) {
         $('tr.value', context).removeClass('truncated');
     }
 
-    $('tr.value td .value', context).text(v);
+    $('tr.value td .value', context).empty().text(v);
+    if (item instanceof Fact && item.isNil()) {
+        $('tr.value td .value', context).wrapInner("<i></i>");
```

Comment:
You could capture the jq object to avoid the double lookup for text+wrapInner (here and `_selectionSummaryAccordian`).

Reply:
Fixed in 4009081.